src/main/java/mate/academy/controllers/DeleteProductFromShoppingCartController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.lib.Injector;
+import mate.academy.service.ProductService;
+import mate.academy.service.ShoppingCartService;
+
+public class DeleteProductFromShoppingCartController extends HttpServlet {
+    private static final Long USER_ID = 1L;
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+    private ShoppingCartService shoppingCartService =
+            (ShoppingCartService) INJECTOR.getInstance(ShoppingCartService.class);
+    private ProductService productService =
+            (ProductService) INJECTOR.getInstance(ProductService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String productId = req.getParameter("id");
+        shoppingCartService.delete(shoppingCartService.get(USER_ID),
```

Comment:
```suggestion
        shoppingCartService.deleteProduct(shoppingCartService.get(USER_ID),
```
пропоную переіменувати метод